kernels/quantized/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+try:
+    from pathlib import Path
+
+    libs = list(
+        Path(__file__)
+        .parent.parent.parent.resolve()
```

Comment:
Ideally the so file should just under the same directory.

Reply:
I see. I copied to the same directory so Path(__file__).parent should be good.